website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -329,10 +329,16 @@ The name and tag of the image (`IMG=<some-registry>/<project-name>:tag`) in both
 
 #### Deploy the operator
 
-For this example we will run the operator in the `default` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
+Per default a new namespace is created with name `<project-name>-system`. For this example we will run the operator in the `memcached-operator` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
 
 ```sh
-$ cd config/default/ && kustomize edit set namespace "default" && cd ../..
+$ cd config/default/ && kustomize edit set namespace "memcached-operator" && cd ../..
```

Comment:
Somewhere along the line, does the `Namespace` in `config/manager/manager.yaml` need to have its `metadata.name` aligned with the `namespace` and `namePrefix` specified in `./config/default/kustomization.yaml`?

Reply:
IMO we should here use the target as well which is : $ make deploy IMG='image:tag' such was we have in the quick-start.

Reply:
@joelanford I am not sure whether it is what you are referring to but with a recent kustomize version (> 3.8.1) I get as output the value specified by set namespace in the resource definition generated by $ kustomize build config/default, here "memcached-operator". kustomize behavior has been changed  here: https://github.com/kubernetes-sigs/kustomize/pull/2742
I did not find a minimal version of kustomize specified as a pre-requisite on the website, possibly something to add.
@camilamacedo86 I am not sure to follow.

Reply:
@fgiloux,

The official versions supported are those ones defined in the Makefile. See: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/memcached-operator/Makefile#L104

Also, in Kubebuilder we already provide a not in the docs to clarifies that. 
```
<aside class="note">
<h1>Versions and Supportability</h1>

Projects created by Kubebuilder contain a Makefile that will install tools at versions defined at creation time. Those tools are:
- [kustomize](https://kubernetes-sigs.github.io/kustomize/)
- [controller-gen](https://github.com/kubernetes-sigs/controller-tools)

The versions which are defined in the `Makefile` and `go.mod` files are the versions tested and therefore is recommend to use the specified versions.

</aside>
```

See the PR: https://github.com/kubernetes-sigs/kubebuilder/pull/1735
I will do a cherry-pick of this PR to update the production doc. We might need to create a PR for our docs as well.

Reply:
@camilamacedo86 it looks good in regards to clarifying the version dependencies. As the kustomize version in Makefile is v3.5.4 we will however get the behavior described where namePrefix is applied to the namespace. What are your views on that?

Reply:
IMO in the tutorial, we could use only the makefile targets. Why do we need to describe here how to deploy the project and ns that is not the default one created by the tool? It shows for me something more advanced that should not be in this doc. So, in POV, we could just to use `make deploy IMG=` and then in the cleanup to use the `make undeploy`

Reply:
@camilamacedo86 OK. I tested with kustomize 3.5.4 and it works well. I will update the PR accordingly a bit later